### PR TITLE
refactor to use new built in nmapper from rdf-store (aka fix/issue-12)

### PR DESCRIPTION
**Pull request by marc-portier**
first commit down this path mostly identifies we need more support from rdf_store first
created issue https://github.com/vliz-be-opsci/py-RDF-store/issues/51 for that

### CODE REVIEW

**Comment by marc-portier** [comment]
NOTE: This relies on the merge of https://github.com/vliz-be-opsci/py-RDF-store/pull/52 back into main

when it is done, we should switch dependencies back to main branch of rdf_store

**Comment by marc-portier** [comment]
just waiting for the merge in https://github.com/vliz-be-opsci/py-RDF-store/pull/52

File: pyproject.toml [hunk 1]
```diff
@@ -39,7 +39,7 @@ sphinx = {version = "^7.0.1", optional = true}
 
 # Poetry has the dependecies groups, but those are not 
 #   compatible with extras, widely used in the python-verse.
-pyrdfstore = {git = "https://github.com/vliz-be-opsci/py-RDF-store.git", rev = "main"}
+pyrdfstore = {git = "https://github.com/vliz-be-opsci/py-RDF-store.git", rev = "fix/issue-51"}
```
Comment (cedricdcc) [inline pyproject.toml]: should we not wait for this to be put into main?

Comment (marc-portier) [reply pyproject.toml]: indeed -- see own comment about that https://github.com/vliz-be-opsci/py-sync-fs-triples/pull/13#issuecomment-2069102417

**Comment by marc-portier** [comment]
@cedricdcc -- had to bump test coverage -- should be ok to merge now

**Review by cedricdcc — APPROVED** [review]

**Pull request merged**